.github/workflows/ci.yml

```diff
@@ -84,6 +84,7 @@ jobs:
       working-directory: ./cmd/xcaddy
       env:
         CGO_ENABLED: 0
+        GOARCH: amd64
```

Comment:
What's this for?

Reply:
Apparently GitHub runners are arm64 :) I figured instead of setting up a matrix of GOARCHes, it's reasonable to just build for ammd64 for the CI artifacts